### ISSUE TITLE
Template Literal support?

### ISSUE THREAD

**Issue by trinonsense** [amilajack/eslint-plugin-compat#216]
I was wondering why Template Literal compatibility support wasn't detected? MDN lists `IE` and `Edge <12` among the browsers not supporting Template Literals, and I had expected `eslint-plugin-compat` to catch it. Thanks!

e.g. 
```
Object.assign({ templateLiteral: `literally templating` });
```

**Comment by amilajack** [comment]
This plugin current supports only JS API's at the moment.

**Comment by ljharb** [comment]
Use babel-preset-env for syntax.
